MyAwesomePlugin/MyAwesomePlugin.csproj

```
@@ -16,4 +16,12 @@
 	<ItemGroup>
 		<ProjectReference Include="..\ArchiSteamFarm\ArchiSteamFarm\ArchiSteamFarm.csproj" ExcludeAssets="all" Private="false" />
 	</ItemGroup>
+	
+	<Target Name="PostBuild" AfterTargets="PostBuildEvent">
+		<Copy
+			SourceFiles="$(TargetPath)"
+			DestinationFolder="$(SolutionDir)ArchiSteamFarm\ArchiSteamFarm\bin\$(Configuration)\$(TargetFramework)\plugins\"
```

See CI failure, I didn't test this but I don't see why it shouldn't work. Current path should be relative to `csproj`, we use the same trick above for project reference.

```suggestion
			DestinationFolder="..\ArchiSteamFarm\ArchiSteamFarm\bin\$(Configuration)\$(TargetFramework)\plugins\"
```